update flows samples to use v1 API (#2633)
knative/docs: docs/eventing/samples/sequence/sequence-terminal/README.md

We are going to create the following logical configuration. We create a

> CronJobSource, feeding events to a [`Sequence`](../../../flows/sequence.md).

Reviewer: CronJobSource -> Ping source?
Maybe lurking in other places too? Perhaps grep for it to get these updated at the same time?

Author: good idea!

> Sequence can then do either external work, or out of band create additional

Reviewer 2: This sentence may be difficult to understand for non native speakers. "Sequence can then either do external work, or create additional events out of band", maybe that will sound a little better but I'm not 100% sure that it means the same thing we're trying to say in this line.

Reviewer: lol, probably because IIRC it was originally written by a non native speaker (I) :)

Reviewer 3: IMHO it doesn't make sense to native speakers either - I really don't understand what this is saying?

Also, avoid using "we" please throughout docs. I see this in PRs still quite a bit, but it's not recommended, as per the style guide: https://developers.google.com/style/person

### Create the PingSource targeting the Sequence

> This will create a PingSource which will send a CloudEvent with
> `{"message": "Hello world!"}` as the data payload every 2 minutes.

Reviewer 3: ```suggestion
This example creates a PingSource that sends a CloudEvent with
`{"message": "Hello world!"}` as the data payload every 2 minutes.
```

### Inspecting the results

You can now see the final output by inspecting the logs of the event-display

> pods. Note that since we set the `PingSource` to emit every 2 minutes, it might
> take some time for the events to show up in the logs.

Reviewer 3: ```suggestion
pods.

**NOTE:** The `PingSource` is set to emit every 2 minutes, so it may
take some time for the events to show up in the logs.
```
